iModelCore/GeomLibs/PublicAPI/Geom/DTriangle3d.h

```diff
@@ -207,7 +194,7 @@ DPoint2dCP pVertex2
 * @param uvw   [out] barycentric coordinates of pPoint relative to T
 * @param dUVWdX   [out] derivatives wrt point.x
 * @param dUVWdY   [out] derivatives wrt point.y
-* @param area  [out] triangle area
+* @param area  [out] signed triangle area

```

Comment:
signed area?! what does it mean?

Reply:
The sign comes from the orientation of the triangle and is negative iff the points are clockwise. This number is readily available, so we return it.  (It's half of the denominator in the Cramer's Rule solution to the system.) I'll add an explanation!